android/src/main/java/io/mosip/tuvali/transfer/Assembler.kt

```diff
@@ -63,8 +64,8 @@ class Assembler(private val totalSize: Int, private val mtuSize: Int = DEFAULT_C
     var missedSeqNumbers = intArrayOf()
     chunkReceivedMarker.forEachIndexed() { i, elem ->
       if (elem != chunkReceivedMarkerByte) {
-        Log.d(logTag, "getMissedSequenceNumbers: adding missed sequence number $i")
-        missedSeqNumbers += i
+        Log.d(logTag, "getMissedSequenceNumbers: adding missed sequence number ${i+1}")
+        missedSeqNumbers = missedSeqNumbers + i + 1
```

Comment:
Can we initialise the sequence number to 1 and start the entire transfer, instead of adding `+1` to the logic?